packages/components/addon/components/hds/table/td.hbs

```diff
@@ -1,3 +1,6 @@
-<td class="hds-table__td hds-typography-body-200 hds-font-weight-regular" ...attributes>
+<td
+  class="hds-table__td hds-typography-body-200 hds-font-weight-regular {{if @alignRight 'hds-table__td--text-right'}}"
```

Comment:
I know some won't like a conditional here; I tried to pass this in the hash (feel like I should have been able to) but it didn't work. I'll try again on Monday if it really bothers folks too much.